code/modules/antagonists/changeling/powers/adrenaline.dm

```diff
@@ -12,21 +12,5 @@
 
 //Recover from stuns.
 /obj/effect/proc_holder/changeling/adrenaline/sting_action(mob/living/user)
-	to_chat(user, "<span class='notice'>Energy rushes through us.[user.lying ? " We arise." : ""]</span>")
-	user.SetSleeping(0)
-	user.SetUnconscious(0)
-	user.SetStun(0)
-	user.SetKnockdown(0)
-	user.reagents.add_reagent("changelingadrenaline", 10)
-	user.reagents.add_reagent("changelinghaste", 2) //For a really quick burst of speed
-	user.reagents.add_reagent("inaprovaline", 3) //let's give another chance to dumb fucks who forget to breathe
-	user.adjustStaminaLoss(-150)
-	user.stuttering = 0
-	user.updatehealth()
-	user.update_stamina()
-	user.resting = 0
-	user.lying = 0
-	user.update_canmove()
-
+	user.do_adrenaline(100, FALSE, 70, 0, TRUE, list("epinephrine" = 3, "changelingmeth" = 10, "mannitol" = 10, "omnizine" = 10, "changelingadrenaline" = 5), "<span class='notice'>Energy rushes through us.</span>")
```

Comment:
```suggestion
	user.do_adrenaline(100, TRUE, 70, 0, TRUE, list("epinephrine" = 3, "changelingmeth" = 10, "mannitol" = 10, "omnizine" = 10, "changelingadrenaline" = 5), "<span class='notice'>Energy rushes through us.</span>")
```

Reply:
30 stamloss regen per mob life cycle. and it's spammable to boot. how is instant recovery from stamcrit + an insanely strong stam regen + the ability to spam it, considered "in the dump"

Reply:
Meh, that and the extra healing effects do make em hard to get robusted, but they are still at a loss crawling around as more people come by to gang em up lethally.
combat intent right click help intent does help but it's fairly clunky to pull off, and combat mode doesn't work well with the pace of your generic ss13 robusting anyway.
Really should do some quick QoL on it.